docs/examples.rst

```diff
@@ -0,0 +1,28 @@
+Examples
+=========
+
+The ``GalSim-Euclid-Like/examples`` directory contains example files for how use the euclidlike module.  
+
+End-to-end demo 
+---------------
+
+:gh-link:`end_to_end_demo.py <examples/end_to_end_demo.py>`
+
+This first demo is the euclidlike-equivalent of  _`demo 13 <https://github.com/GalSim-developers/GalSim/blob/main/examples/demo13.py>`_ in ``GalSim``. This demo uses the Euclid-like PSF, WCS, and background noise to produce a realistic scene of galaxies and stars as observed from a Euclid-like Telescope. 
+
+** Features introduced in the Python file**:
```

Comment:
These asterisks are not doing what was intended (if you look at the html it produces)

Reply:
Addressed.